bears/c_languages/CPPCheckBear.py

```diff
@@ -19,6 +19,13 @@ class CPPCheckBear:
     """
 
     LANGUAGES = {"C", "C++"}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'https://groups.google.com/forum/#!forum/coala-devel'}
+    MAINTAINERS = {'Lasse Schuirmann', 'Fabian Neuschmidt', 'Mischa Kr\xfcger'}
```

Comment:
isn't this the coala devs as well? Could we maybe default it to  the AUTHORS if not given?